src/controller.js

```diff
@@ -134,6 +134,32 @@ var API = {
     })
     req.end()
   },
+  getNewTweets: function (cb) {
+    const options = {
+      hostname: 'api.twitter.com',
+      port: 443,
+      path: format('/2/tweets/search/stream'),
+      method: 'GET',
+      headers: {
+        'Authorization': format('Bearer {0}', twitterBearToken)
+      }
+    }
+    const req = https.request(options, (res) => {
+      console.log(`statusCode: ${res.statusCode}`)
+      let data = ''
+      res.on('data', (chunk) => {
```

Comment:
looks good. I think there are two things missing for this to work e2e. 
1. Here, we need to read the chunk and if it's a valid tweet, pass it on to the tweet parser for processing. See https://github.com/twitterdev/real-time-tweet-streamer/blob/master/server/server.js#L122 for an example. 
2. We need to start the stream once when the server starts up. We don't want to have to manually call it from postman.

Reply:
@kgao @caiyuan1222 @wangmeng255 I made these changes with latest push. It starts the stream when the server starts and passes any new tweets to the parser, which will publish them if they are valid. To test, pull the latest, run the server (npm run start) then make a tweet mentioning @SaveTheDose with location on. Then refresh the dashboard (aka.ms/commoncircle) and the new dot should show up in the location of the tweet